Packs/Phishing/Scripts/PhishingDedupPreprocessingRule/PhishingDedupPreprocessingRule.py

```diff
@@ -258,15 +258,17 @@ def format_similar_incident(incident, similairy):
             }
 
 
-def create_new_incident_low_similarity(existing_incident, new_incident, similarity):
+def create_new_incident_low_similarity(existing_incident, similarity):
     message = '# This incident is not a duplicate of an existing incident.\n'
-    formatted_incident = format_similar_incident(existing_incident, similarity)
-    message += tableToMarkdown("Most similar incident found", formatted_incident)
-    message += 'The threshold for considering 2 incidents as duplicate is a similarity ' \
-               'of {:.1f}%.\n'.format(SIMILARITY_THRESHOLD * 100)
-    message += 'Therefore these 2 incidents will not be considered as duplicate and the current incident ' \
-               'will remain active.\n'
-    return_entry(message)
+
+    if similarity > SIMILARITY_THRESHOLD - CLOSE_TO_SIMILAR_DISTANCE:
+        formatted_incident = format_similar_incident(existing_incident, similarity)
+        message += tableToMarkdown("Most similar incident found", formatted_incident)
+        message += 'The threshold for considering 2 incidents as duplicate is a similarity ' \
+                   'of {:.1f}%.\n'.format(SIMILARITY_THRESHOLD * 100)
+        message += 'Therefore these 2 incidents will not be considered as duplicate and the current incident ' \
+                   'will remain active.\n'
+        return_entry(message)
 
```

Comment:
I think it's also good to return a message  for "else" case which can be the same message as other cases when no incident found

Reply:
the first line said that no duplicate incident found

Reply:
the return is within the if, don't think you return it

Reply:
good point, fixed